README.md

```
@@ -111,6 +124,15 @@ Learn how API Gateways work through practical scenarios and real-world examples,
 - [Rate limiting](#rate-limiting) and traffic control
 - Protection for **GraphQL**, **JSON**, and **XML** APIs against malicious inputs.
 
+### **AI Gateway**
+- [LLM Gateway](https://www.membrane-api.io/api-key-sharing-for-ai-and-llm-models.html)
+- [Securely Share LLM API Keys](https://www.membrane-api.io/api-key-sharing-for-ai-and-llm-models.html)
+
+### **AI Gateway**
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)